### PR TITLE
Change generating of graphs

### PR DESCRIPTION
**Pull request by Duxo**
Now graphs can generate different colored points.
Its because of transparency of data which are shown on graph.

### CODE REVIEW

File: src/main/java/hudson/plugins/report/genericchart/ChartModel.java [hunk 1]
```diff
@@ -139,4 +140,10 @@ public String getResultWhiteList() {
         return resultsWhiteList;
     }
 
+    public String getPointColor(boolean isInRangeOfWhiteListed) {
+        if (isInRangeOfWhiteListed) {
+            return ColorChanger.shiftColorBy(chartColor, 64, 64, 32);
```
Comment (judovana) [inline src/main/java/hudson/plugins/report/genericchart/ChartModel.java]: Can you  specify what kind of shift do you expect? why 64 64 32 ?-)

File: src/main/java/hudson/plugins/report/genericchart/PropertiesParser.java [hunk 2]
```diff
@@ -37,6 +37,8 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 import java.util.stream.Stream;
+import java.util.Random;
+
```
Comment (judovana) [inline src/main/java/hudson/plugins/report/genericchart/PropertiesParser.java]: Random? why?

File: src/main/java/hudson/plugins/report/genericchart/PropertiesParser.java [hunk 3]
```diff
@@ -189,9 +195,10 @@ public ChartPointsWithBlacklist getReportPointsWithBlacklist(Job<?, ?> job, Char
                         .filter((o) -> o.isPresent())
                         .map(o -> o.get())
                         .map(s -> new ChartPoint(
-                                run.getDisplayName(),
-                                run.getNumber(),
-                                extractValue(s)))
+                        run.getDisplayName(),
+                        run.getNumber(),
+                        extractValue(s),
+                        chart.getPointColor(pointsInRangeOfwhitelisted.contains(run.getDisplayName()))))
```
Comment (judovana) [inline src/main/java/hudson/plugins/report/genericchart/PropertiesParser.java]: Origina code indention  seemed to be better.

File: src/main/resources/hudson/plugins/report/genericchart/GenericChartProjectAction/floatingBox.jelly [hunk 4]
```diff
@@ -22,14 +22,18 @@
                         label: "${chart.title}",
                                 fillColor: "${chart.color}",
                                 strokeColor: "${chart.color}",
-                                pointColor: "${chart.color}",
-                                pointStrokeColor: "#fff",
+                                pointStrokeColor: "#808080",
                                 pointHighlightFill: "#fff",
-                                pointHighlightStroke: "rgba(180,180,180,1)",
+                                pointHighlightStroke: "rgba(0,0,0,1)",
```
Comment (judovana) [inline src/main/resources/hudson/plugins/report/genericchart/GenericChartProjectAction/floatingBox.jelly]: Why this change?

Comment (Duxo) [reply src/main/resources/hudson/plugins/report/genericchart/GenericChartProjectAction/floatingBox.jelly]: StrokeColor is changed to #808080 so if it would stay without change the color difference between highlighted and normal strokeColor would  be small. Which is bad because of readability
